### ISSUE TITLE
[1.4.x] Calling module gives another error

### ISSUE THREAD

**Issue by Portugao** [Guite/MostGenerator#751]
```
Neither the property "album" nor one of the methods "album()", "getalbum()"/"isalbum()" or "__call()" exist and have public access in class "Symfony\Component\Form\FormView" in @MUMUImageModule/Album/viewQuickNav.html.twig at line 15
```

**Comment by Portugao** [comment]
This a the correct error message; sorry.

**Comment by Guite** [comment]
Fixed in 14c6a21c66c791e7312a6d927051a4a4ad6365f1
